### PR TITLE
install.sh: Add message on error exit and simplify installation command

### PR DESCRIPTION
**Pull request by supechicken**
## Changes
- Added an error trap that prints a message when the script exits on error occurs
- Remove silent flag in `curl` so that network errors can be shown
- Makes the bootstrap command more straightforward by removing the `exec` hack and source `~/.bashrc` in the parent shell

### Run the following to get this pull request's changes locally for testing.
```shell
bash <(curl -L https://raw.githubusercontent.com/supechicken/chromebrew/install_sh_error_handling/install.sh) && . ~/.bashrc
```

### CODE REVIEW

**Comment by supechicken** [comment]
Umm... Should I take care of shellcheck suggestions? Those `|| exit 1` suggestions sounds unnecessary.

**Comment by uberhacker** [comment]
Shellcheck is complaining @supechicken.

**Comment by supechicken** [comment]
> Shellcheck is complaining @supechicken.

Read my previous comment...

**Comment by uberhacker** [comment]
> > Shellcheck is complaining @supechicken.
> 
> Read my previous comment...

Can we disable rules in the .yml config?

**Comment by supechicken** [comment]
> Can we disable rules in the .yml config?

Done

**Comment by Zopolis4** [comment]
SC2164 is supposed to be disabled by set -e, which I thought we had.

**Comment by supechicken** [comment]
> SC2164 is supposed to be disabled by set -e, which I thought we had.

I think you are right, as I replaced `set -e` with `set -E`. Let me add it back...

**Comment by Zopolis4** [comment]
If we source ~/.bashrc in the install command then we don't need to tell people to do it manually.

**Comment by supechicken** [comment]
> If we source ~/.bashrc in the install command then we don't need to tell people to do it manually.

Removed `source ~/.bashrc` prompt

File: install.sh [hunk 1]
```diff
@@ -12,6 +12,9 @@ echo_success() { echo -e "\e[1;32m${*}${RESET}" >&1; } # Use Green for success m
 echo_intra() { echo -e "\e[1;34m${*}${RESET}" >&1; } # Use Blue for intrafunction messages.
 echo_out() { echo -e "\e[0;37m${*}${RESET}" >&1; } # Use Gray for program output.
 
+# Print a message before exit on error
+trap "echo_error 'An error occured during the installation :/'" ERR
```
Comment (uberhacker) [inline install.sh:16]: Add color red?

Comment (uberhacker) [reply install.sh:16]: Nevermind.  That's what `echo_error` does already. :+1:

File: README.md [hunk 2]
```diff
@@ -57,10 +57,10 @@ You will need a Chromebook with developer mode enabled. To do so, select your de
 
 Open a VT-2 terminal session with <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>-></kbd> and login with the `chronos` user and password if set [above](#set_passwd). *(if you are unable to do this, please have a second look at the prerequisites and make sure your Chromebook is in developer mode)*
 
-Then run the installation script below:
+Then download and run the installation script:
```
Comment (uberhacker) [inline README.md:60]: We're just running it, right?  If we weren't redirecting it to bash then it would be considered a  download.

**Review by satmandu — APPROVED** [review]

**Comment by satmandu** [comment]
Please merge master and modify line 60 as per @uberhacker 's request so we can merge this?

File: README.md (hunk 2 above)
Comment (supechicken) [reply README.md]: Change reverted

**Comment by satmandu** [comment]
Please merge master again, as we just updated this for M123...

**Review by uberhacker — APPROVED** [review]

**Pull request merged**